packages/SwingSet/src/controller/startNodeSubprocess.js

```diff
@@ -13,6 +13,12 @@ export function makeStartSubprocessWorkerNode(
     ).pathname;
     const args = nodeOptions ? [...nodeOptions] : [];
     if (profileVats.includes(vatID)) {
+      // Enable profiling with a 1 microsecond sampling interval,
+      // saving results to a file in the working directory
+      // with a portable name derived from the vat name
+      // and in particular replacing any colons with dashes.
+      // cf. ../kernel/vat-loader/manager-subprocess-node.js
+      // https://github.com/Agoric/agoric-sdk/blob/18d561f1b95755e58ca691b26938fc249618018d/packages/SwingSet/src/kernel/vat-loader/manager-subprocess-node.js#L32-L34
```

Comment:
I don't like putting a permalink to a particular old (or soon to be old) version into current sources. I appreciate this link for explanation as a reviewer of the PR. I think it belongs in the PR notes instead of the source. But please let me know if I'm missing something.

Reply:
Yeah, I don't have a great solution for this problem... the _intent_ is to reference a sibling as it exists during introduction of the comments (in this case to point out that colons are expected because other code is adding them) that will still make sense in the future if that sibling code is moved, changed, or deleted. Without a permalink, the risk is that such changes will undermine comprehension of this comment unless the archaeologist remembers to blame-chase the comment lines back to their first commit and then traverse the repository within the corresponding tree. Not that we expect severe consequences here, but I've adopted a general practice of including both kinds of reference just in case.

Reply:
Oh, I missed the redundancy. Good enough! Resolving.